Extensions/outbox.js

```diff
@@ -177,7 +177,9 @@ XKit.extensions.outbox = new Object({
 
 		var form_key = XKit.interface.form_key();
 
-		$(document).on("click", "#ask_button", function() {
+		$(document).on("click", ".post-form--asks .ask-button", function() {
+
+			var $form = $(this).closest('form');
```

Comment:
.closest() will travel up the DOM tree from the element until it finds the selector, while .parent only travels up one level. This requires you to do parent().parent().parent() etc. which makes it easy to break after tumblr changes something. And .parents() is less efficient than .closest().